collect_app/src/main/java/org/odk/collect/android/tasks/InstanceUploaderTask.java

```diff
@@ -74,6 +74,7 @@ public class InstanceUploaderTask extends AsyncTask<Long, Integer, InstanceUploa
     // Custom submission URL, username and password that can be sent via intent extras by external
     // applications
     private String completeDestinationUrl;
+    private String referrer = "org.odk.collect.android";
```

Comment:
Sorry for the back and forth, but I just caught this. Why do we set this to `"org.odk.collect.android"`? Surely it only gets used if `completeDestinationUrl` is set (which it's set with) so we can just leave it unassigned. 

Just want to make sure we're only logging in cases where the custom URL is being used here and that I'm not missing something.

Reply:
You are right it was redundant.